### ISSUE TITLE
Alien Hominid Invasion Beta (1764450)

### ISSUE THREAD

**Issue by hplewis** [ValveSoftware/Proton#5309]
# Compatibility Report
- Name of the game with compatibility issues: Alien Hominid Invasion Beta
- Steam AppID of the game: 1764450

## System Information
- GPU: GTX 760
- Driver/LLVM version: nvidia ~470.82.00~ updated to 470.86
- Kernel version: 5.14.16-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): [system info
](https://gist.github.com/hplewis/53793bad8a0a02a712655fba23c9d2a9)
- Proton version: 6.3-7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1764450.log](https://github.com/ValveSoftware/Proton/files/7530592/steam-1764450.log)

## Symptoms

When trying to start the game, nothing loads and the following error message is shown. After pressing "OK", the game closes.
![screenshot](https://user-images.githubusercontent.com/8607347/141539364-537cf868-3e1b-4d44-80c9-fdd6a739d8d9.png)

## Reproduction

1. Start Alien Hominid Invasion Beta on Proton 6.3-7
2. Wait for error message (about 3 seconds)
3. Press OK
4. Game closes

**Comment by hplewis** [comment]
I did see a few updates for related pieces of my system, but they are pretty recent updates (last couple days). I'll update just to be sure and then check off that part.

**Comment by hplewis** [comment]
Just fully updated my system (nvidia drivers are legacy 470xx due to my card being old).

New game crash log: 
[steam-1764450.log](https://github.com/ValveSoftware/Proton/files/7530646/steam-1764450.log)

New system info: [system info](https://gist.github.com/hplewis/aeae4a6318a8a21bfd876801a0666d01)

Same kernel 5.14.16-arch1-1

**Comment by hplewis** [comment]
Steam system info doesn't show nvidia drivers there, but xorg seems to show me running nvidia. Maybe because I'm running legacy [470xx from the AUR](https://aur.archlinux.org/packages/nvidia-470xx-utils/)?

**Comment by kisak-valve** [comment]
Hello @hplewis, looking at your system information, it looks like you've configured your system to use the nvidia gpu, but don't have the video driver's 32 bit userspace libraries installed. Can you check if you have `lib32-nvidia-470xx-utils` installed?

**Comment by hplewis** [comment]
You're right. I forgot that one when switching to legacy. Just installed it, but seem to be encountering the same issue.
New system info: [system info](https://gist.github.com/hplewis/f06ce2927263cc6a481b302753b3c3fa)
New crash report (similar to before): 
[steam-1764450.log](https://github.com/ValveSoftware/Proton/files/7531204/steam-1764450.log)

Error message/symptoms/etc are the same as before.
